src/components/Card.jsx

```diff
@@ -1,14 +1,20 @@
+import { Link } from "react-router-dom";
 const Card = (props) => {
+  const { id } = props;
+  //   console.log(id);
+
   return (
-    <div className="card">
-      <h4>{props.topic}</h4>
-      <h3>{props.title}</h3>
-      <img src={props.article_img_url} />
-      <p>Written Bye: {props.author}</p>
-      <p>{props.comment_count}</p>
-      <p>Votes: {props.votes}</p>
-      <p>On: {new Date(props.created_at).toDateString()}</p>
-    </div>
+    <Link to="/article" state={{ article_id: id }}>
+      <section className="card">
+        <h4>{props.topic}</h4>
+        <h3>{props.title}</h3>
+        <img src={props.article_img_url} />
+        <p>Written By: {props.author}</p>
+        <p>{props.comment_count}</p>
+        <p>Votes: {props.votes}</p>
```

Comment:
I'd recommend using icons to indicate what the numbers represent too. Right now, the user wouldn't be able to tell what the number for comment count is as there is no icon 👀